server/index.js

```diff
@@ -167,20 +137,6 @@ export class Server extends BaseServer {
       }
     })
 
-    for (let name in opts) {
```

Comment:
Why we check that no unknown options are given after server construction? Can we do this check before?

We will not be able to use a reporter to log an error, but do we really need a formatted error when a server cannot be started?

Reply:
> We will not be able to use a reporter to log an error, but do we really need a formatted error when a server cannot be started?

Nope. We can even remove it. Let’s users use TypeScript if they want extra checks.